src/main/java/tech/gruppone/stalker/server/repositories/PlacePositionRepository.java

```diff
@@ -10,11 +11,18 @@
 public interface PlacePositionRepository extends ReactiveCrudRepository<PlacePositionDao, Long> {
 
   @Query("SELECT id, ST_ASGEOJSON(position) AS positionGeoJson FROM PlacePosition WHERE id = :id")
-  Mono<PlacePositionDao> findById(final Long id);
+  Mono<PlacePositionDao> findById(@Param("id") final Long id);
 
   // returns number of affected rows (should be 0 or 1)
   @Modifying
   @Query(
       "INSERT INTO `PlacePosition`(`id`, `position`) VALUES (:id, ST_GeomFromGeoJSON(:rawPositionJson))")
```

Comment:
Qui la query si può lasciare?

Reply:
questo è l'unico caso in cui le query derivate non sono applicabili perché dobbiamo invocare una funzione che converte GeoJSON a WKB (il POLYGON di mysql)